custom_completions.go

```diff
@@ -143,10 +143,6 @@ func (c *Command) initCompleteCmd(args []string) {
 				fmt.Fprintln(finalCmd.OutOrStdout(), comp)
 			}
 
-			if directive >= shellCompDirectiveMaxValue {
-				directive = ShellCompDirectiveDefault
-			}
```

Comment:
This change is unrelated but I just wanted to get it in at the same time.
It allows a program to use extra directives if it wants to provide its own shell script, instead of using Cobra's.